cli/ballerina-cli/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -91,15 +91,16 @@ OPTIONS
             Additional build options to be passed to the GraalVM native image.
 
        --parallel
-            Enable parallel execution of tests. A test should satisfy the following conditions to be
-            executed in parallel:
-                - Test function should be isolated.
-                - If it is a data provider test,
-                    - The data provider of a test function should be isolated.
-                    - Test function parameters should be read-only type.
-                - Respective set-up, and tear-down functions (before, after, before-each, after-each, before-groups,
-                  after-groups) of the test function should be isolated.
-            Parallel test execution is an experimental feature which supports only a limited set of functionality.
+            Enable parallel execution of tests. A test must satisfy the following
+            conditions to be executed in parallel:
+                - Test function must be isolated.
+                - If a data provider is attached to a test function,
+                    - The data provider function must be isolated.
+                    - The types of the test function parameters must be subtypes of readonly.
+                - Respective set-up, and tear-down functions (before, after, before-each,
```

Comment:
```suggestion
                - Corresponding set-up and tear-down functions (before, after, before-each,
```

Shall we use single quotes or backticks to emphasize the function names?

Reply:
Shall we remove the function names from this description.